Data/Graph.hs

```diff
@@ -23,53 +23,76 @@
 -- Maintainer  :  libraries@haskell.org
 -- Portability :  portable
 --
--- A version of the graph algorithms described in:
+-- = Finite Graphs and Strongly Connected Components
 --
---   /Structuring Depth-First Search Algorithms in Haskell/,
---   by David King and John Launchbury.
+-- The @'Graph'@ type is an adjacency list representation of a finite, directed
+-- graph with @Int@ vertices.
+--
+-- The @'SCC'@ type represents a strongly-connected component of a graph.
```

Comment:
Web link to an explanation of what that means.

Reply:
Added link to wikipedia.